app/controllers/concerns/voteable_controller.rb

```diff
@@ -0,0 +1,11 @@
+module VoteableController
+  extend ActiveSupport::Concern
+
+  def self.included(base)
+    base.helper_method :parents_vote
+  end
+
+  def parents_vote(parent)
+    @vote = Vote.find_by(user_id: current_user, voteable_id: parent)
```

Comment:
unnecessarily setting an ivar here. I'd also probably find some way to put a method on `Vote` to having something like this in your views `current_user.votes.find_by(voteable: parent)`. there might be more performant ways to go about this, but this should get the job done without requiring a method on the controllers.

Reply:
cool - thanks - closing this PR - have copied this comment to Issue https://github.com/gingin77/hackernews_clone/issues/30